lib/modules/apostrophe-seo-doc-type-manager/index.js

```diff
@@ -18,22 +18,23 @@ module.exports = {
         {
           name: 'seoRobots',
           label: 'Robots Tag',
+          htmlHelp: '<i class="fa fa-info-circle"></i> <a href=https://moz.com/learn/seo/robots-meta-directives data-toggle="tooltip" target="new_window" title="Learn More About Robots Meta Tag">Learn More</a>',
```

Comment:
This is more complex than I was thinking, and also requires the click. We should start people out with some description immediately, linking part of the description out as with normal rich text. So 1) the icon should go (not a standard practice for Apostrophe at the moment), 2) the "Robots tag" label should be made more lay-friendly (e.g., Search engine indexing setting"), and 3) provide a bit more than "learn more" (maybe "Learn more about these options.") to provide context.